src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -607,6 +612,10 @@ int fx_muxer_t::parse_args_and_execute(
         known_opts.push_back(_X("--depsfile"));
         known_opts.push_back(_X("--runtimeconfig"));
     }
+    if (mode == host_mode_t::muxer)
```

Comment:
Isn't this going to be misleading for the other modes if `--help` says you can pass `--fx-version`, but then parsing that option fails because it wasn't added here?